sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Interface/IKeyVaultDefault.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Azure.Security.KeyVault.Cryptography.Interface
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public interface IKeyVaultDefault
```

Comment:
I'm not sure what this is needed for but if it's necissary it shouldn't be public

Reply:
Whenever in doubt I fall back to v1 and check what it means.
So when I saw bunch of string defaults in v1, I wanted to make it a type and abstract individuals into a parent type.
Will make it non-public.